events/events-team/content/new-contributor-workshops/workspace-setup.md

```diff
@@ -12,7 +12,7 @@ Make sure all Beginning Contributors have a workspace setup for hacking on Kuber
 
 ### Slides
 
-Clear and large download commands to copy into terminal
+Clear and large download commands to copy onto the terminal.
```

Comment:
I feel like "into the" would be proper here, but that's just a feeling.  Prepositions...the hardest part of most languages?